### ISSUE TITLE
[feature request] Support more commands on Android TV Integration

### ISSUE THREAD

**Issue by splattner** [unfoldedcircle/bug-reports#92]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Based on the [HomeAssistant Documentation for the Android TV Remote](https://www.home-assistant.io/integrations/androidtv_remote/#remote), a Android TV Remote seems to support more commands:

```
Navigation:

DPAD_UP
DPAD_DOWN
DPAD_LEFT
DPAD_RIGHT
DPAD_CENTER
BUTTON_A
BUTTON_B
BUTTON_X
BUTTON_Y
```
```
Volume Control:

VOLUME_DOWN
VOLUME_UP
VOLUME_MUTE
MUTE
```
```
Media Control:

MEDIA_PLAY_PAUSE
MEDIA_PLAY
MEDIA_PAUSE
MEDIA_NEXT
MEDIA_PREVIOUS
MEDIA_STOP
MEDIA_RECORD
MEDIA_REWIND
MEDIA_FAST_FORWARD
```
```
TV Control:

0
1
2
3
4
5
6
7
8
9
DEL
ENTER
CHANNEL_UP
CHANNEL_DOWN
F1
F2
F3
F4
F5
F6
F7
F8
F9
F10
F11
F12
TV
PROG_RED
PROG_GREEN
PROG_YELLOW
PROG_BLUE
```
```
Other:

BUTTON_MODE
EXPLORER
MENU
HOME
INFO
GUIDE
TV_TELETEXT
CAPTIONS
DVR
MEDIA_AUDIO_TRACK
SETTINGS
SEARCH
ASSIST
 ```

Currently the R2 Android TV Integration does only support some of them. Specially the ones for TV Control / Numbers I am missing.

I know, the [mediaplayer entity ](https://github.com/unfoldedcircle/core-api/blob/main/doc/entities/entity_media_player.md) does not seems to support a number pad, so in order to implement some of the additional supported commands, the media_player entity also needs some work. See https://github.com/unfoldedcircle/feature-and-bug-tracker/issues/91

**Comment by splattner** [comment]
oh, and maybee open source the android tv integration as well? Then people could also contribute.

**Comment by zehnm** [comment]
The just released uc-api 0.9.0 supports more media-player features and a "simple command" option for all device commands not covered by a feature: https://github.com/unfoldedcircle/core-api/releases/tag/v0.9.0-beta

Next steps:
- [x] enhance Node.js & Python API wrapper libraries (ETA today)
- [x] enhance firmware (core, UI, web-configurator) (ETA this week)
- [x] enhance Android TV integration (open issue: distinguish TV vs streaming device to expose supported commands)

**Comment by zehnm** [comment]
The next release supports the new media-player features and simple commands.

New Android TV integration version will soon be released in beta channel.

**Comment by zehnm** [comment]
Included in the just released beta firmware 1.7.9 and will be part of the next public release.

To update an already configured Android TV media-player entity with the new features: click "Reload" at the bottom of the available entities pane in the web-configurator. This will load the new entity features and commands from the integration and update the configured entity.

See https://github.com/unfoldedcircle/integration-androidtv/blob/main/docs/command_mapping.md for new commands and how to test your device with additional Android TV remote keycodes. Unfortunately every Android TV device is different and only supports a subset of keycodes.

**Comment by zehnm** [comment]
Included in release 1.7.12
